retrofit/WebServices/src/main/java/com/example/aperez/retrofittest/mvp/view/ResponseView.java

```diff
@@ -35,12 +42,35 @@ public void setCards(List<Image> images) {
         recyclerView.setAdapter(adapter);
     }
 
+    public void initializeLoader() {
+        getActivity().getSupportLoaderManager().initLoader(0, null, new LoaderManager.LoaderCallbacks<Cursor>() {
```

Comment:
The initLoader should be in the activity. Its not necessary implements a callback. 
See this example:  getLoaderManager().initLoader(R.id.my_loader, null, presenter);
An remember destroy the loader.